Athos/ONNXCompiler/.gitignore

```diff
@@ -0,0 +1,7 @@
+models/
```

Comment:
I think if u do this, then the folders themselves will not get checked in and on a fresh clone, the user would have to create the folder themselves. Is this handled elsewhere - like whereever this is used in an automated manner (like for eg the scripts would create this folder automatically on detecting no such folder)? Or else you can also checkin a .gitignore file inside the folder and git add that .gitignore.

Reply:
yes. the script starts with creating the folder first if it does not exist. Though there is issue with tests where this is not handled. but I think we can ignore it for now.